composer.json

```diff
@@ -4,7 +4,7 @@
     "type": "luya-kickstarter",
     "license":"MIT",
     "require": {
-        "luyadev/luya-module-admin" : "^4.0",
+        "luyadev/luya-module-admin" : "dev-master as 4.2.0",
```

Comment:
I don't think this is required for now, as we will release 4.2 soon. otherwise we might forget to revert to 4.2 on release